test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
```

Comment:
why @command and why in a sub-directory _cmd_? _basic_ and _coolstore_ are also running a "bunch" of commands. Is the plan to create a feature for each command individually? The problem I see here is that a lot of commands will require a running Minishift instance. Provisioning a Minishift instance per command will take probably too much time. The version command is a bit special here since it requires no running VM. 

Either way, unless we have a concrete plan with the _cmd_ sub-directory I would argue that there is no need yet to create new directories.